### DISCUSSION TITLE
Bigger topMargin on first page

### DISCUSSION THREAD

**Discussion by Miurath** [eclipse-birt/birt#961] [General]
Hi, 
it seems there is no possibility to have a two diffrent topMargines in the same report, if i only use one masterpage. I can not use multiple masterpages, since I can not predict how big the elements are and how many will be printed on the first page, before a pageBreak.

Since the topMargine is set in the beforeFactory() I am not able to change it later on during the report.

Is there some workaround for having a diffrent topMargin on the first page?

**Comment by hvbtup** [comment]
What I do is (works with the PDF emitter):
You can use e.g. a negative margin for the the very first object on the first page.
This comes handy if you want to have a different header on the first page:
Uncheck "Show header on first page" for the master page.
Create a grid to be used as a "replacement header" for the first page in the body (not in the master page). Use a negative margin for that grid.
Probably you can use a similar approach.

**Comment by Miurath** [comment]
Thank you, it worked as expected :)
